README.md

```diff
@@ -1,2 +1,23 @@
-# agora
-The place where SpartanZ make decisions
+# Welcome to the SpartanZ Agora
+
+Yes, we know, the agora is an Athenian thing, but metaphors are meant to be broken, right?
+
+This is the place where we make organization-wide decisions. The Spartanz organization has no leader nor hierarchy, therefore everything must be decided collectively. 
+
+The present repository is used for that very purpose. Every decision we make is materialized as a change of this repository's content (ie. a pull request) and consequently, at every point in time, the content of this repository's master branch constitutes the "law" that rules our organization.
+
+## Making changes
+
+All [active members](TODO: definition of membership) of the organization are equal and everyone is equally entitled to propose changes in the way the organization works. In all circumstances, change is achieved using a mix of issues and pull-requests to the present repository.
```

Comment:
`active members` will be important because you don't want a bunch of "armchair" participants making decisions; you have to be a "doer" and have "skin in the game" to be involved.